src/Peering/Peering.Test/ScenarioTests/CreateNewExchangePeeringTests.cs

```diff
@@ -59,15 +59,5 @@ public void TestNewExchangePeeringPipe()
         {
             TestController.NewInstance.RunPowerShellTest(this.logger, "Test-NewExchangePeeringPipe");
         }
-
-        /// <summary>
-        /// The test new exchange peering pipe two connections.
-        /// </summary>
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
-        public void TestNewExchangePeeringPipeTwoConnections()
```

Comment:
@rileymckenna any reason this test is being removed?

Reply:
These were duplicate tests. or they aren't supported by the back end right now.

Reply:
Just to confirm, when you say that a test isn't supported in the back end right now, does that mean it previously worked and you were able to record the test previously, but now does not work when trying to record the test against the live server?